src/search/pdbs/pattern_database.cc

```diff
@@ -280,16 +280,28 @@ bool PatternDatabase::is_goal_state(
     return true;
 }
 
-size_t PatternDatabase::hash_index(const vector<int> &state) const {
+size_t PatternDatabase::hash_index_of_concrete_state(const vector<int> &state) const {
     size_t index = 0;
     for (size_t i = 0; i < pattern.size(); ++i) {
         index += hash_multipliers[i] * state[pattern[i]];
     }
     return index;
 }
 
+size_t PatternDatabase::hash_index_of_projected_state(const State &projected_state) const {
```

Comment:
shouldn't this (for consistency with `hash_index_of_concrete_state`) also just take a `const vector<int>` as parameter?

Reply:
I'll leave that open for now. If we actually get rid of `ProjectedTask`, then this needs to be changed anyway.